angr/analyses/decompiler/structured_codegen/c.py

```diff
@@ -914,12 +916,22 @@ def c_repr_chunks(self, indent=0, asexpr=False):
                 yield indent_str, None
             else:
                 yield " ", None
-            yield "{", brace
-            yield "\n", None
-            if node is not None:
-                yield from node.c_repr_chunks(indent=indent + INDENT_DELTA)
-            yield indent_str, None
-            yield "}", brace
+
+            omit_if_brace = (isinstance(node, CStatements) and len(node.statements) == 1) or isinstance(node, CContinue)
```

Comment:
This condition is not checking if it should `omit_if_brace`. This condition is more so checking `is_single_stmt_if`. You would then use this variable in an if-stmt below like you do which would show that this condition is tied to omitting the brace. Change the name here to be better.